src/test/java/com/google/api/generator/gapic/composer/goldens/EchoClient.golden

```diff
@@ -63,7 +63,20 @@ import javax.annotation.Generated;
  *
  * <p>To customize credentials:
  *
+ * <pre>{@code
```

Comment:
Wasn't this already checked in? Can we sync it so that the diffs from prior PRs don't show up here?

Reply:
Will do.